src/services/auth/oidc/loginAttempts/types.ts

```diff
@@ -0,0 +1,72 @@
+import { OptionsWithRql, PagedResultWithPager } from '../../../types';
+import { FindAllIterator } from '../../../helpers';
+
+export interface LoginAttemptsService {
```

Comment:
I took some liberties in formatting the documentation in what I thought in a slightly better way here. Please have a lookie at the result in the SDK and see what you think.